### PR TITLE
docs: Set up locales for existing translations

### PR DESCRIPTION
**Pull request by dcwalk**
This PR arises from https://github.com/cosmos/gaia/issues/977: 
- moves the existing translations into their own folder
- sets up locales for  `es` `en-US` `ko` `zh-CN`
- adds a bit of detail to the DOCS_README (borrowed from cosmos-sdk docs): https://github.com/cosmos/cosmos-sdk/blob/master/docs/DOCS_README.md

### CODE REVIEW

File: docs/es/README.md [hunk 1]
```diff
@@ -1,22 +1,16 @@
-<!--
-parent:
-  order: false
-layout: home
--->
-
```
Comment (okwme) [inline docs/es/README.md:6]: @dcwalk did you confirm this doesen't break the vue build? i had thought these comments were necessary for some part of the build process

Comment (dcwalk) [reply docs/es/README.md:6]: Confirmed just now by building, a index is generated in with the page contents:
Screenshot_2021-12-09_09-40-32 

I removed frontmatter here to match zh and ko READMEs, but happy to add to all if preference is for that. 

If you are referring to the "order: false" line (which is mentioned in [cosmos theme readme](https://github.com/cosmos/vuepress-theme-cosmos#file-configuration)), it doesn't seem like it is needed when setting up translated folders, none of them are added to the English version navigation.

Comment (okwme) [reply docs/es/README.md:6]: 👍

**Review by okwme — APPROVED** [review]

**Comment by okwme** [comment]
ah looks like there's some conflicts, can you resolve those and i'll merge?

**Comment by dcwalk** [comment]
resolved @okwme!

File: docs/DOCS_README.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ parent:
 
 If you want to open a PR on Gaia to update the documentation, please follow the guidelines in the [`CONTRIBUTING.md`](https://github.com/cosmos/gaia/tree/main/CONTRIBUTING.md)
 
+## Internationalization
+
+- Translations for documentation live in a `docs/<locale>/` folder, where `<locale>` is the language code for a specific language. For example, `zh` for Chinese, `ko` for Korean, `es` for Spanish, etc.
```
Comment (yaruwangway) [inline docs/DOCS_README.md:12]: `docs/<locale>/` ->  `docs/translations/<locale>/`

Comment (dcwalk) [reply docs/DOCS_README.md:12]: This PR removes the `/translations/` subfolder as that actually doesn't allow translated text to render on built docs!

Comment (yaruwangway) [reply docs/DOCS_README.md:12]: ha ,ok!

**Review by yaruwangway — APPROVED** [review]

**Comment by dcwalk** [comment]
This is out of date with main again, I'll ff and resolve the conflict then merge!

**Pull request merged**